From fbbb37b5f725f94ed6d16c1cb0d245d435cef6c8 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Sun, 07 Apr 2019 20:30:41 +0300
Subject: [PATCH] [SW-0] make: add specs to header search path

This paves the way to full decoupling from u-boot (GPL project)

Change-Id: Ia987503470130cc3c47c9e36bdcafeeab692264b
---

diff --git a/Makefile b/Makefile
index f3e40a1..a7f8ef4 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,8 @@
 CC = $(CROSS_COMPILE)gcc
 OBJCOPY = $(CROSS_COMPILE)objcopy
-CFLAGS := -g -nostdinc -fno-pic -fno-builtin
+CFLAGS += -I$(dir $(BR2_EXTERNAL))/specs/$(HL_DEV)/asic_reg
+CFLAGS += -I$(dir $(BR2_EXTERNAL))/embedded/specs
+CFLAGS += -g -nostdinc -fno-pic -fno-builtin
 CFLAGS += -D__KERNEL__ -D__UBOOT__ -D__ARM__ -march=armv8-a -D__LINUX_ARM_ARCH__=8
 CFLAGS += -isystem ../../host/opt/ext-toolchain/lib/gcc/aarch64-linux-gnu/6.4.1/include
 CFLAGS += -I ../uboot-custom/include
